doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,65 @@
+[[renaming-ha]]
+== Renaming Database-Only Appliances
+
+Renaming database-only appliances in a high availability setup must be performed in a specific order to maintain database replication.
+
+//Are all steps in the appliance console menu?
+
+. Confirm and note the hostnames of the active database (primary) appliance and the standby database appliance in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
+//Is there an easier way than logging into the appliance console of one appliance and verifying if "Local Database Server" says "running (primary)" or "running (standby)"?
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. Stop the failover monitor on the non-database {product-title_short} appliances:
+.. In the appliance console menu, select *Configure Application Database Failover Monitor*. 
+.. Select *Stop Database Failover Monitor*.
+. Reset the standby by deleting the database:
+//Configure Database Replication > Configure Server as Standby? Then ??
```

Comment:
I think it would be easier to just stop the database service on the standby here.

Reintroducing it will remove and recreate the data so there's not need to delete the database here.